nendo_server/handler/nendo_actions_handler.py

```diff
@@ -290,64 +291,179 @@ def create_docker_action(
+        run_without_target: bool,
+        max_track_duration: float,
+        max_chunk_duration: float,
         env: Optional[dict] = None,
         func_timeout: int = 0,
         **kwargs, # these are the parameters for the action script
     ) -> str:
+        # get queues
         gpu = gpu and self.server_config.use_gpu
         cpu_queue, gpu_queue = self.worker_manager.get_user_queues(user_id)
         queue = gpu_queue if gpu else cpu_queue
+        
+        # assign job and container name
         letters_and_digits = string.ascii_letters + string.digits
         rnd_string = "".join(random.choice(letters_and_digits) for i in range(8))
         job_id = action_name.replace(" ", "_") + "_" + rnd_string
         container_name = container_name if exec_run is True else job_id
-        command = self._generate_command(
-            user_id,
-            job_id,
-            **kwargs,
-        )
-        job = queue.enqueue(
-            dockerized_func,
-            user_id,
-            image,
-            script_path,
-            command,
-            plugins,
-            self.nendo_config,
-            self.server_config,
-            gpu,
-            container_name,
-            exec_run,
-            replace_plugin_data,
-            env,
-            func_timeout,
-            description=action_name,  # TODO improve description?
-            job_id=job_id,  # use custom job id (for in-job status reporting)
-            job_timeout="72h",  # TODO: make this configurable?
-            result_ttl="172800",  # 2 days retention of completed jobs
+        
+        target_id = kwargs.get("target_id", "")
+        track_or_collection = self.nendo_instance.get_track_or_collection(
+            target_id,
         )
-        job.meta["action_name"] = action_name
-        job.meta["parameters"] = pprint.pformat(kwargs)
         target = {}
-        if "target_id" in kwargs:
-            target_id = kwargs["target_id"]
-            track_or_collection = self.nendo_instance.get_track_or_collection(
-                target_id,
-            )
-            if type(track_or_collection) == NendoTrack:
-                target.update({
-                    "target_type": "track",
-                })
-            else:
-                target.update({
-                    "target_type": "collection",
-                })
+        if track_or_collection is not None:
+            target.update({
+                "target_type": (
+                    "track" if isinstance(track_or_collection, NendoTrack) else
+                    "collection",
+                )
+            })
             target.update({
                 "target_id": target_id,
             })
-        job.meta["target"] = target
-        job.save_meta()
-        return job.id
+        target_collections = []
+        skipped_tracks = []
+        # apply chunking, if configured and applicable
+        if (self.server_config.chunk_actions is True and
```

Comment:
could be shortened to: 

```suggestion
        if (self.server_config.chunk_actions and not run_without_target and gpu)
```